spec/requests/atmosphere/api/virtual_machine_flavors_spec.rb

```diff
@@ -166,23 +166,23 @@
           expect(response.size).to eq 1
           first = response.first
           expect(first['active']).to be_truthy
-          expect(first['compute_site_id']).to eq active_cs.id
+          expect(first['tenant_id']).to eq active_t.id
         end
 
         context 'reqiurements specified' do
 
           it 'returns one flavor' do
-            tmpl = create(:virtual_machine_template, appliance_type: at, compute_site: cs, state: 'active')
+            tmpl = create(:virtual_machine_template, appliance_type: at, tenant: t, state: 'active')
             get api("virtual_machine_flavors?cpu=1&memory=1024&hdd=5&appliance_type_id=#{at.id}", user)
             flavors = fls_response
             expect(flavors.size).to eq 1
           end
         end
 
-        context 'filter for cs specified' do
-          it "applies compute site filtering" do
-            tmpl = create(:virtual_machine_template, appliance_type: at, compute_site: cs2, state: 'active')
-            get api("/virtual_machine_flavors?appliance_type_id=#{at.id}&compute_site_id=#{cs2.id + 1}", user)
+        context 'filter for tenant specified' do
+          it "applies tenant filtering" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.